docker/prepare-docker

```diff
@@ -15,7 +15,7 @@ build_docker() {
     docker build \
         --squash \
         --no-cache \
-        -t perconalab/ps-build:${SOURCE_IMAGE//[:\/]/-} \
```

Comment:
@Sudokamikaze we will use 'ps-build' not 'fb-*'

Reply:
Removed deps install and rebased them to jenkins-pipeline repo